020_app_L44/020_app_L44_unsafe.py

```
@@ -0,0 +1,108 @@
+import jsonpickle
+import requests
+
+from flask import Flask, render_template
+from lxml import etree
+from werkzeug.utils import redirect
+
+app = Flask(__name__)
+
+@app.route("/rce/<string:payload>")
+def definite_rce(payload: str) -> None:
+    subprocess.run(payload, shell=True)
```

Ruff (0.14.1)

`subprocess` call with `shell=True` identified, security issue

(S602)